action.yml

```diff
@@ -12,4 +12,4 @@ runs:
       shell: bash
       run: ${{ github.action_path }}/scripts/main.sh
       env:
-        NPM_TOKEN: ${{ inputs.npm-token }}
+        YARN_NPM_AUTH_TOKEN: ${{ inputs.npm-token }}
```

Comment:
Apparently Yarn uses a different environment variable name. It doesn't seem to be documented, but I found references to it on GitHub. Testing with this one now.

Reply:
It looks like this is documented at the top of this page: https://yarnpkg.com/configuration/yarnrc. Although it could be easily missed. I suppose we could keep `NPM_TOKEN` and then have `main.sh` set `yarn config set npmAuthToken $NPM_TOKEN`. But I think this is probably okay.